test/screenshotter/ss_data.json

```diff
@@ -4,6 +4,7 @@
     "Baseline": "http://localhost:7936/test/screenshotter/test.html?m=a+b-c\\cdot d/e",
     "BasicTest": "http://localhost:7936/test/screenshotter/test.html?m=a",
     "BinomTest": "http://localhost:7936/test/screenshotter/test.html?m=\\dbinom{a}{b}\\tbinom{a}{b}^{\\binom{a}{b}+17}",
+    "Cases": "http://localhost:7936/test/screenshotter/test.html?m=f%28a%2Cb%29%3D\\begin{cases}a%2B1%26\\text{if%20}b\\text{%20is%20odd}\\\\a%26\\text{if%20}b%3D0\\\\a-1%26\\text{otherwise}\\end{cases}",
```

Comment:
In the short term it might be easiest to add a button to index.html that shows the JSON string in an alert or something.  I wouldn't worry about it for this PR.  Here's the string without the URL encoded chars:
```
f(a,b)=\\begin{cases}a+1&\\text{if }b\\text{ is odd}\\\\a&\\text{if }b=0\\\\a-1&\\text{otherwise}\\end{cases}
```